packages/framework-integration-tests/integration/aws/utils.ts

```diff
@@ -199,6 +199,11 @@ export async function graphQLClient(authToken?: string): Promise<ApolloClient<No
   return new ApolloClient({
     cache: cache,
     link: link,
+    defaultOptions: {
+      query: {
+        fetchPolicy: 'no-cache',
+      },
+    },
```

Comment:
Is this intended?

Reply:
Yes, if we have the cache enabled in tests, they will fail (we do some retries)